src/app/shared/filter-lang-pipe/filter-lang.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'filterLang',
+  pure: false
+})
+export class FilterLangPipe implements PipeTransform {
+  transform(langArr, lang) {
+    return langArr.filter((value) => value.code === lang);
+  }
+}
```

Comment:
please create a more general pipe for filtering f.e -> https://github.com/VadimDez/ngx-filter-pipe/blob/master/src/app/shared/ngx-filter-pipe/ngx-filter.pipe.ts
so it can handle more than langArr.filter....
!!!!!also angular team removed filter pipe because of performance reasons so if you can do filtering in .ts file it would be better